### ISSUE TITLE
Back cover not working

### ISSUE THREAD

**Issue by 1marc1** [asciidoctor/asciidoctor-pdf#1804]
Hi team,

Not sure if I am doing something wrong, but I cannot get the back cover to work. Here is the minimal example that I created:

```
marc@home:/tmp/test$ asciidoctor-pdf --version
Asciidoctor PDF 1.5.3 using Asciidoctor 2.0.10 [https://asciidoctor.org]
Runtime Environment (ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux-gnu]) (lc:UTF-8 fs:UTF-8 in:UTF-8 ex:UTF-8)
marc@home:/tmp/test$ ll
total 140
drwxrwxr-x  2 marc marc   4096 Oct 22 13:07 ./
drwxrwxrwt 38 root root   4096 Oct 22 13:02 ../
-rwxr-xr-x  1 marc marc 123570 Oct 22 13:03 backcover.png*
-rw-rw-r--  1 marc marc     76 Oct 22 13:03 file.adoc
-rw-rw-r--  1 marc marc     88 Oct 22 13:04 theme.yml
marc@home:/tmp/test$ cat theme.yml 
extends: default
base:
  font-color: #ff0000
cover:
  back-image: image:backcover.png[]
marc@home:/tmp/test$ cat file.adoc 
= Test

== Back cover

This document tests to see if there is a back cover.
marc@home:/tmp/test$ asciidoctor-pdf -a pdf-theme=theme.yml file.adoc --trace -v
marc@home:/tmp/test$ 
```

As you can see, there is no further output from running ```asciidoctor-pdf```, but the image is not showing up in the resulting PDF file either (see attached).
[file.pdf](https://github.com/asciidoctor/asciidoctor-pdf/files/5419602/file.pdf)

Thank you for any suggestions you may have.

Marc.

**Comment by mojavelinux** [comment]
Defining a cover using the theme file is not a feature of 1.5.3. See https://github.com/asciidoctor/asciidoctor-pdf/blob/v1.5.x/docs/theming-guide.adoc The feature is only available in the upcoming 2.0.0 release. Make sure you are using the theming guide on the v1.5.x branch when using 1.5.3.

**Comment by 1marc1** [comment]
Ah... that explains. Thank you Dan! Any idea when 2.0.0 will be ready?

**Comment by mojavelinux** [comment]
Working hard on it every day. As soon as I get the fix for #789 integrated, I'm going to release the first alpha. We'll go from there.
